### PR TITLE
mesoscale explorer UI, advanced measurement for interactive labels

### PR DESCRIPTION
**Pull request by corredD**
# Description
Snapshot description advanced Markdown command for highlight when mouse over link text :
    - `#` other snapshots with a given key `[...](#key)`
    - `i` highlight a protein with a given NAME `[...](iNAME)`
    - `g` highlight a group with a given group type and group name `[...](ggrouptype.groupname)`
    - `h` URLs with a given link `[...](http...)`
Snapshot description panel window size and text can be resized and hidden with new icons
Added measurement control, added label are clickable to load a different snapshot with given #key
Added selection description panel

This is the first PR, selection, protein description and coloring PR will follow.

### CODE REVIEW

File: src/mol-plugin-ui/skin/base/components/viewport.scss [hunk 1]
```diff
@@ -173,15 +173,17 @@
...
     left: 0;
     top: $control-spacing + $row-height;
     padding: (0.66 * $control-spacing) $control-spacing;
 
-    max-height: 7 * $row-height;
-    overflow: hidden;
-    overflow-y: auto;
-
-    max-width: 400px;
+    // max-height: 7 * $row-height;
```
Comment (arose) [inline src/mol-plugin-ui/skin/base/components/viewport.scss:181]: no commented-out code please

File: src/mol-plugin-ui/controls/icons.tsx [hunk 2]
```diff
@@ -133,6 +133,8 @@ const _GetApp = <svg width='24px' height='24px' viewBox='0 0 24 24'><path d='M19
 export function GetAppSvg() { return _GetApp; }
 const _HelpOutline = <svg width='24px' height='24px' viewBox='0 0 24 24'><path d='M11 18h2v-2h-2v2zm1-16C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8zm0-14c-2.21 0-4 1.79-4 4h2c0-1.1.9-2 2-2s2 .9 2 2c0 2-3 1.75-3 5h2c0-2.25 3-2.5 3-5 0-2.21-1.79-4-4-4z' /></svg>;
 export function HelpOutlineSvg() { return _HelpOutline; }
+const _Tour = <svg width='24px' height='24px' viewBox='0 0 24 24'><path d='M21 4H7V2H5v20h2v-8h14l-2-5zm-6 5c0 1.1-.9 2-2 2s-2-.9-2-2 .9-2 2-2 2 .9 2 2' /></svg>;
```
Comment (arose) [inline src/mol-plugin-ui/controls/icons.tsx:136]: just to be sure, are the icons you added from https://material-ui.com/components/material-icons/? if not please put them in a different section and mention their license

**Comment by arose** [comment]
How do you add a selection description? Or will that come in a later PR?

Why expose behavior settings?

File: src/mol-plugin-ui/controls/icons.tsx (hunk 2 above)
Comment (corredD) [reply src/mol-plugin-ui/controls/icons.tsx:136]: good point I will check

**Comment by corredD** [comment]
yes the description will come in after this PR. I realized after that this PR included the widget and the state string. Should I remove it ? 
For the behavior controller, Its the only way I found to expose a control for the camera click behavior to switch between center on/off. So that each snapshot could have a different mouse behavior. Some people like click to center and focus, other just click and center.

**Comment by corredD** [comment]
I am closing and separating the changes. will do a PR only on the markdown snapshot description.

**Pull request closed without merging**